packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -171,7 +178,8 @@ export function getTickerNameByBlockchain(chain: Blockchain) {
 }
 
 export function getApiUrlByBlockchainType(chain: Blockchain): string {
-    return isTestnet(chain) ? CROSSMINT_STG_URL : CROSSMINT_PROD_URL;
+    const result = isTestnet(chain) ? CROSSMINT_STG_URL : CROSSMINT_PROD_URL;
+    return result;
```

Comment:
nit: we can just return the result instead of splitting the line
```suggestion
    return isTestnet(chain) ? CROSSMINT_STG_URL : CROSSMINT_PROD_URL;
```